news/2023/2023-07-16-osulazer-updates-july-16-2023.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "osu!(lazer) Updates: July 16, 2023"
+date: 2023-07-16 12:00:00 +0000
+---
+
+Nothing's better than a brand-new osu!(lazer) update on a hot summer day...
```

Comment:
```suggestion
Nothing's better than a brand-new osu!(lazer) update on a hot summer day.
```
this is the largest nitpick of the century, but triple periods usually infer uncertainty or hesitation. we love the osu!(lazer). there is no uncertainty, hot summer lazer updates are GOOD, and thus use one period